packages/styled-base/types/tests.tsx

```diff
@@ -82,22 +82,15 @@ const Canvas1 = styled('canvas', {
 interface PrimaryProps {
   readonly primary: string
 }
-/**
- * @desc
- * This function accepts `InnerProps`/`Tag` to infer the type of `tag`,
- * and accepts `ExtraProps` for user who use string style
- * to be able to declare extra props without using
- * `` styled('button')<ExtraProps>`...` ``, which does not supported in
- * styled-component VSCode extension.
- * If your tool support syntax highlighting for `` styled('button')<ExtraProps>`...` ``
- * it could be more efficient.
- */
-const Button2 = styled<'button', PrimaryProps>('button')`
-  fontsize: ${5}px;
+
+const Button2 = styled('button')<PrimaryProps>`
```

Comment:
I have updated the test to use this overload instead. I didn't have any issues with the styled components extension in VSCode. All works great

Reply:
Docs have also been updated to be inline with this style